buildroot-external/kernel/v6.6.y/hassos.config

```diff
@@ -43,6 +43,11 @@ CONFIG_SQUASHFS=y
 CONFIG_SQUASHFS_XATTR=y
 CONFIG_SQUASHFS_LZ4=y
 CONFIG_BTRFS_FS=m
+CONFIG_NTFS_FS=m
+CONFIG_NTFS3_FS=m
+CONFIG_EXFAT_FS=m
+CONFIG_FUSE_FS=m
+CONFIG_XFS_FS=m
 
```

Comment:
```suggestion
```

It's very unlikely that FUSE will be needed for the use case that this PR aims to address - i.e. mounting a disk with less common (in the Linux world) filesystem. 

Similarly XFS, with its deployment rather in the enterprise segment. Which I'd prefer removing for one another reason, the module binary is quite big and space becomes a problem especially on the x86-64 platform.

Reply:
Ok, that sounds ok for me, thanks